tests/jest/config/platform-properties.json

```diff
@@ -119,30 +119,33 @@
   "modal.enforce-hcmcompliance.label": "Enforce policy",
   "modal.inform-hcmcompliance.heading": "Inform policy",
   "modal.inform-hcmcompliance.label": "Inform policy",
-  "modal.automation-hcmcompliance.heading": "Edit policy violation automation",
   "modal.enable.description": "Are you sure you want to enable this policy?",
   "modal.disable.description": "Are you sure you want to disable this policy?",
   "modal.inform.description": "Are you sure you want to change the remediation action of this policy to inform? You must manually remediate the policy violations.",
   "modal.enforce.description": "Are you sure you want to enforce this policy? The policy makes any necessary changes to the applied cluster to remediate the violations.",
-  "modal.ansible.automation.description": "Modify an existing provider credentials, or create a new one to automate your selected policy and violation. You can also schedule the automation to run once, continuously, or disable for now.",
-  "modal.ansible.launch.connection": "Launch to connection URL",
-  "modal.ansible.credential.selection.title": "Provider connection",
-  "modal.ansible.credential.selection.placeholder": "Select Ansible Credential",
-  "modal.ansible.no.credential": "Ansible Credential is required to create Ansible Job, create your console here:",
-  "modal.ansible.launch.createCredential": "Create a new provider credentials",
-  "modal.ansible.jobTemplates.selection.title": "Job template",
-  "modal.ansible.jobTemplates.selection.placeholder": "Select Ansible Job Template",
-  "modal.ansible.jobTemplates.editor.title": "Ansible Job Template Extra Variables",
-  "modal.ansible.jobTemplates.editor.placeholder": "Input Ansible Job Template Extra Variables Here",
-  "modal.ansible.no.jobTemplates.info": "Failed to get Ansible Job Template information, please check related Ansible Tower and Ansible Job Template setting",
-  "modal.ansible.scheduleMode.title": "Schedule automation",
-  "modal.ansible.scheduleMode.info": "Choose how often you want this automation to run. You can also choose to manually run it right now, which will save your change as is.",
-  "modal.ansible.scheduleMode.manual.title": "Manual run",
-  "modal.ansible.scheduleMode.manual.info": "Manual make this automation immediately run once. Once it runs, it will be set to disabled.",
-  "modal.ansible.scheduleMode.once.title": "Run once mode",
-  "modal.ansible.scheduleMode.once.info": "When this policy is violated, the automation will run one time. Once it runs, it will be set to disabled.",
-  "modal.ansible.scheduleMode.disabled.title": "Disable automation",
-  "modal.ansible.scheduleMode.disabled.info": "Automation won't run until you change this setting.",
+  "ansible.automation.heading.create": "Create policy violation automation",

```

Comment:
Please make the same updates as suggested in the previous file (`/nls/platform.properties`)